functions-python/batch_process_dataset/tests/test_helpers.py

```diff
@@ -0,0 +1,162 @@
+import hashlib
```

Comment:
[question]: Why is this test file in the batch_process_dataset folder?

Reply:
`batch_process_dataset` was the one using the helper functions the most (`batch_datasets` only used 1) -- so i thought i'd test it as part of it. However, i do agree that it doesn't seem like the best approach. Would moving these tests under the `helpers` be better?

Reply:
Unless there is an issue running tests or packaging the functions; moving the test inside the helpers' folder will be easier to find.